app/helpers/application_helper.rb

```diff
@@ -1,2 +1,12 @@
 module ApplicationHelper
-end
+
+  # Returns the full title on a per-page basis.
+  def full_title(page_title = '')
```

Comment:
```suggestion
  def full_title page_title = ''
```
k cần dùng ngoặc cũng được